src/cli.rs

```diff
@@ -33,75 +33,78 @@ pub fn clear_screen() {
+            .arg(Arg::with_name("command")
+                     .help("Command to execute")
+                     .multiple(true)
+                     .required(true))
+            .arg(Arg::with_name("extensions")
+                     .help("Comma-separated list of file extensions to watch (js,css,html)")
+                     .short("e")
+                     .long("exts")
+                     .takes_value(true))
+            .arg(Arg::with_name("path")
+                     .help("Watch a specific directory")
+                     .short("w")
+                     .long("watch")
+                     .number_of_values(1)
+                     .multiple(true)
+                     .takes_value(true))
+            .arg(Arg::with_name("clear")
+                     .help("Clear screen before executing command")
+                     .short("c")
+                     .long("clear"))
+            .arg(Arg::with_name("restart")
+                     .help("Restart the process if it's still running")
+                     .short("r")
+                     .long("restart"))
+            .arg(Arg::with_name("signal")
+                     .help("Send signal to process upon changes, e.g. SIGHUP")
+                     .short("s")
+                     .long("signal")
+                     .takes_value(true)
+                     .number_of_values(1)
+                     .value_name("signal"))
+            .arg(Arg::with_name("debug")
+                     .help("Print debugging messages to stderr")
+                     .short("d")
+                     .long("debug"))
+            .arg(Arg::with_name("filter")
+                     .help("Ignore all modifications except those matching the pattern")
+                     .short("f")
+                     .long("filter")
+                     .number_of_values(1)
+                     .multiple(true)
+                     .takes_value(true)
+                     .value_name("pattern"))
+            .arg(Arg::with_name("ignore")
+                     .help("Ignore modifications to paths matching the pattern")
+                     .short("i")
+                     .long("ignore")
+                     .number_of_values(1)
+                     .multiple(true)
+                     .takes_value(true)
+                     .value_name("pattern"))
+            .arg(Arg::with_name("no-vcs-ignore")
+                     .help("Skip auto-loading of .gitignore files for filtering")
+                     .long("no-vcs-ignore"))
+            .arg(Arg::with_name("postpone")
+                     .help("Wait until first change to execute command")
+                     .short("p")
+                     .long("postpone"))
+            .arg(Arg::with_name("poll")
+                     .help("Forces polling mode")
+                     .long("force-poll")
+                     .value_name("interval"))
+            .arg(Arg::with_name("once").short("1").hidden(true))
+            .get_matches();
```

Comment:
Though the `-k` option is superceded by `-s`, I don't want to remove support for it since we are post 1.0. Let's make `-k` act like `-s SIGKILL`.